magicctapipe/scripts/lst1_magic/semi_automatic_scripts/check_MAGIC_runs.py

```diff
@@ -0,0 +1,88 @@
+#This script allows us to get information about every MAGIC run ID (and subruns) in files used for common data analysis (MAGIC1, MAGIC2, #LST1). You can also run the script using Jupyter Notebook.
+
+#The MAGIC files that can be used for analysis are located here:
+#/fefs/onsite/common/MAGIC/data/M{tel_id}/event/Calibrated/{year}/{month}/{day}
+
+#In this path, 'tel_id' refers to the telescope ID, which must be either 1 or 2. 'Year,' 'month,' and 'day' specify the date.
+
+#In the first step, we have to load a dataframe that contains information about the date, the name of the source, and the range of MAGIC #runs. The file in file_path was generated using the spreadsheet (Common MAGIC LST1 data) from the following link:
+
+#https://docs.google.com/spreadsheets/d/1Tya0tlK-3fuN6_vXOU5FwJruis5vBA9FALAPLFHOfBQ/edit#gid=1066216668
```

Comment:
better to remove the link. It includes private MAGIC and LST information, while github is public. The link is protected from random access, but still better to avoid giving its link directly